todocore/src/test/java/com/knighttodo/todocore/integration/DayTodoResourceIntegrationTest.java

```diff
@@ -331,38 +329,4 @@ public void findDayTodosByDayId_shouldReturnExistingTodo_whenIdIsCorrect() throw
                 .andExpect(jsonPath(buildJsonPathToLength()).value(2));
     }
 
-    @Test
-    public void updateIsReady_shouldReturnOk_shouldMakeIsReadyTrue_whenDayTodoIdIsCorrect() throws Exception {
-        Day day = dayRepository.save(DayFactory.dayInstance());
-        DayTodo dayTodo = dayTodoRepository.save(DayTodoFactory.dayTodoWithDayInstance(day));
-        ExperienceResponse experienceResponse = DayTodoFactory.experienceResponseInstance(dayTodo.getId());
-
-        when(restTemplate.postForEntity(anyString(), any(), eq(ExperienceResponse.class)))
-                .thenReturn(new ResponseEntity<>(experienceResponse, HttpStatus.OK));
-
-        mockMvc.perform(put(buildUpdateTodoReadyBaseUrl(day.getId(), dayTodo.getId()))
-                .param(PARAM_READY, PARAMETER_TRUE))
-                .andExpect(status().isOk())
-                .andExpect(jsonPath(buildJsonPathToDayId()).isNotEmpty())
-                .andExpect(jsonPath(buildJsonPathToExperience()).isNotEmpty())
-                .andExpect(jsonPath(buildJsonPathToReadyName()).value(true));
-
-        assertThat(dayTodoRepository.findByIdAlive(dayTodo.getId()).get().isReady()).isEqualTo(true);
-    }
-
-    @Test
-    public void updateIsReady_shouldReturnOk_shouldMakeIsReadyFalse_whenDayTodoIdIsCorrect() throws Exception {
-        Day day = dayRepository.save(DayFactory.dayInstance());
-        DayTodo dayTodoWithReadyTrue = dayTodoRepository.save(DayTodoFactory.dayTodoWithDayReadyInstance(day));
-        ExperienceResponse experienceResponse = DayTodoFactory.experienceResponseInstance(dayTodoWithReadyTrue.getId());
-
-        when(restTemplate.postForEntity(anyString(), any(), eq(ExperienceResponse.class)))
-                .thenReturn(new ResponseEntity<>(experienceResponse, HttpStatus.OK));
-
-        mockMvc.perform(put(buildUpdateTodoReadyBaseUrl(day.getId(), dayTodoWithReadyTrue.getId()))
-                .param(PARAM_READY, PARAMETER_FALSE))
-                .andExpect(status().isOk());
-
-        assertThat(dayTodoRepository.findByIdAlive(dayTodoWithReadyTrue.getId()).get().isReady()).isEqualTo(false);
-    }
```

Comment:
These tests shouldn't be removed, you call Controller method through the mockMvc object, you should leave it because the functionality wasn't removed. So tests should be present. Just comment them for now with I'll fix them later.